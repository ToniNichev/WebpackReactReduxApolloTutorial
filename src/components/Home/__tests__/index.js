import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import MyComponent from '../index.js';

describe('Testing Greetings component', () => {
  it('renders as expected!!!', () => {
    const wrapper = shallow(
      <MyComponent />
    );
   expect(toJson(wrapper)).toMatchSnapshot();
 });
});