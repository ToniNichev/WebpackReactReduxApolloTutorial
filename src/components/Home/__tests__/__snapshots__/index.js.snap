// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Greetings component renders as expected!!! 1`] = `
<div>
  <div>
    This is my home section!
  </div>
</div>
`;
